docs/binary.md

```diff
@@ -514,13 +514,15 @@ The `ColorSequence` type is stored as a `u32` indicating how many `ColorSequence
 
 `ColorSequenceKeypoint` is a struct composed of the following fields:
 
-| Field Name        | Format              | Value                                          |
-|:------------------|:--------------------|:-----------------------------------------------|
-| Time              | `f32`               | The time value of the `ColorSequenceKeypoint`  |
-| Color             | [`Color3`](#Color3) | The color value of the `ColorSequenceKeypoint` |
-| Envelope (unused) | `f32`               | n/a; serialized, but not used                  |
-
-Note that `Color` is **not** subject to interleaving like a normal `Color3`. When multiple `ColorSequence` values are present, they are stored in sequence with no transformation or interleaving. Two `ColorSequence` values
+| Field Name        | Format              | Value                                              |
+|:------------------|:--------------------|:---------------------------------------------------|
+| Time              | `f32`               | The time value of the `ColorSequenceKeypoint`      |
+| ColorR            | `f32`               | The red component of the keypoint's color value.   |
```

Comment:
It's probably fine to just use `R`/`G`/`B` instead of `ColorR` etc.; it's awkward to read with the current proposed names.

Reply:
I went with ColorR since it matched UDim2.ScaleX *that I now realize isn't specified that way*. Okay then. I guess it's also similar to the Ray fields. Could be `R`, `Color R`, or even `Red`. Any preference? I'm not really sure what the spec is going for when listing field names. They seem to be halfway between short descriptions and identifiers.

Reply:
There's a lack of standardization for this spec so realistically anything would be fine, but I would probably go with `R` since that's what we have Color3's components listed as.

We'll probably have to go through and change a few other types to match this format but I think it's a better way to document things, so I'll put in the work (assuming you don't).